### PR TITLE
i18n(zh-cn): translate directus.md

### PR DESCRIPTION
**Pull request by HashCookie**
#### What kind of changes does this PR include?

- Translated content

#### Description

-  Translate directus.md

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/directus.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Directus 与 Astro
+description: 使用 Directus 作为 CMS 向你的 Astro 项目添加内容
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/directus.mdx:3]:
```suggestion
description: 使用 Directus 作为 CMS 为你的 Astro 项目添加内容
```

File: src/content/docs/zh-cn/guides/cms/directus.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+type: cms
+stub: true
+service: Directus
+i18nReady: true
+---
+
+[Directus](https://directus.io/)是一个后端即服务，可用于为你的 Astro 项目托管数据和内容。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/directus.mdx:10]:
```suggestion
[Directus](https://directus.io/) 是一个 BaaS（后端即服务），可用于为你的 Astro 项目托管数据和内容。
```

**Review by 100gle — CHANGES_REQUESTED** [review]
可直接点击 Suggestions 修改即可

**Review by 100gle — APPROVED** [review]
LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**